fapolicy_analyzer/glade/trust_file_details.glade

```diff
@@ -2,18 +2,21 @@
 <!-- Generated with glade 3.38.2 -->
 <interface>
   <requires lib="gtk+" version="3.22"/>
-  <object class="GtkBox" id="trustFileDetails">
+  <!-- n-columns=3 n-rows=3 -->
```

Comment:
In the glade file you seem to still have extra columns that are not being used for anything.  You should change the grid properties to get rid of those.
![image](https://user-images.githubusercontent.com/17859684/143040641-fa91abdb-753f-4af1-a624-77866661b57b.png)

Reply:
Fixed